lib/MusicBrainz/Server/Email.pm

```diff
@@ -483,12 +483,16 @@ sub send_editor_report {
     my $reported_user = $opts{reported_user};
     my $subject = 'Editor ' . $reported_user->name . ' has been reported by ' . $reporter->name;
     my $reason = $MusicBrainz::Server::Form::User::Report::REASONS{$opts{reason}};
-
+    my $reporter_tolink = uri_escape_utf8($reporter);
+    my $reported_user_tolink = uri_escape_utf8($reported_user);
     my $body .= <<EOF;
 $subject for the following reason:
 
 “$reason”
 
+Links to related accounts :
```

Comment:
It think it was better without this line. The links are already individually labeled.